### PR TITLE
ARTART review

### PR DESCRIPTION
**Pull request by marco-tiloca-sics**
This PR addresses the ARTART review archived at https://mailarchive.ietf.org/arch/msg/core/2jCRvv6eWQxP7RMu0WPulmVHi4E/

### CODE REVIEW

File: draft-ietf-core-oscore-edhoc.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ This triggers the EDHOC execution at the server, which replies with a 2.04 (Chan
...
 
-After this exchange takes place, and after successful verifications as specified in the EDHOC protocol, the client and server can derive an OSCORE Security Context, as defined in {{Section A.1 of I-D.ietf-lake-edhoc}}. After that, they can use OSCORE to protect their communications as per {{RFC8613}}.
+After this exchange takes place, and after successful verifications as specified in the EDHOC protocol, the client and server can derive an OSCORE Security Context, as defined in {{Section A.1 of I-D.ietf-lake-edhoc}}. After that, they can use OSCORE to protect their communications as per {{RFC8613}}. Note that the EDHOC Connection Identifier C_R is used as OSCORE Sender ID of the client (see {{Section A.1 of I-D.ietf-lake-edhoc}}). Therefore, C_R is transported in the 'kid' field of the OSCORE Option of the OSCORE Request (see {{Section 6.1 of RFC8613}}).
```
Comment (cabo) [inline draft-ietf-core-oscore-edhoc.md]: (small improvement)

```suggestion
After this exchange takes place, and after successful verifications as specified in the EDHOC protocol, the client and server can derive an OSCORE Security Context, as defined in {{Section A.1 of I-D.ietf-lake-edhoc}}. After that, they can use OSCORE to protect their communications as per {{RFC8613}}. Note that the EDHOC Connection Identifier C_R is used as the OSCORE Sender ID of the client (see {{Section A.1 of I-D.ietf-lake-edhoc}}). Therefore, C_R is transported in the 'kid' field of the OSCORE Option of the OSCORE Request (see {{Section 6.1 of RFC8613}}).
```

**Review by cabo — APPROVED** [review]
(Small optional wordsmithing comment.)
I'm assuming the Section 3 text change is the same as in the GENART, to which I made a few more wordsmithing comments.

**Review by rikard-sics — APPROVED** [review]
Looks good to me

**Comment by gselander** [comment]
Merging as no objections are noted.

**Pull request merged**